imports/components/currency/index.js

```diff
@@ -27,17 +39,29 @@ export const currencySizeCalc = (baseHeadingSize:string) => {
   ReducedHeadingSize = `h${reducedHeadingResult}`;
 };
 
+export const textTheme = (
+  theme?: string,
+): string => {
+  if (theme === "light") {
+    return "text-light-primary";
+  }
+
+  return "text-dark-primary";
+};
+
 const Currency = ({
   amount,
```

Comment:
I'd like to be able to accept a number as well as a string. This can be done by adjusting the `getFoo` calls to something like this: `getDollars(`${amount}`, roundCurrency)` which will cast the string or number to a string.